BoxContentSDK/BoxContentSDK/Operations/BOXAPIOperation.m

```diff
@@ -302,9 +313,15 @@ - (void)executeOperation
 
         if (self.error == nil && ![self isCancelled])
         {
-            self.connection = [[NSURLConnection alloc] initWithRequest:self.APIRequest delegate:self];
-            BOXLog(@"Starting %@", self);
-            [self startURLConnection];
+            //FIXME: remove shouldUseSessionTask and this condition after switching
+            //completely from NSURLConnection to NSURLSessionTask
```

Comment:
maintain support for both nsurlsessiontask and nsurlconnection at this base class to allow switching few groups of operations to nsurlsessiontask at a time